crates/shielder-scheduler-server/src/storage/dynamo_db.rs

```diff
@@ -0,0 +1,475 @@
+use aws_sdk_dynamodb::{
+    error::SdkError,
+    operation::{describe_table::DescribeTableError, put_item::PutItemError},
+    types::{
+        AttributeDefinition, AttributeValue, GlobalSecondaryIndex, KeySchemaElement, KeyType,
+        Projection, ProjectionType, ProvisionedThroughput, ScalarAttributeType,
+    },
+    Client,
+};
+use chrono::{DateTime, Utc};
+
+use crate::storage::{RequestStatus, ScheduledRequest, StorageError, StorageInterface};
+
+const TABLE_NAME: &str = "ScheduledRequests";
```

Comment:
In DynamoDB, table name should be unique within region. Since scheduler is now designed to run one instance per chain, we should parametrize table name with some unique, deterministic suffix, e.g `ScheduledRequests_<chain_id>`, where `chain_id` is a unique chain identifier.